<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MealTabInsertWidget</class>
 <widget class="QWidget" name="MealTabInsertWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="label_dinner">
       <property name="text">
        <string>Dinner</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="push_br">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QLabel" name="label_lunch">
       <property name="text">
        <string>Lunch</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label_breakfast2">
       <property name="text">
        <string>2nd breakfast</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QLabel" name="label_tea">
       <property name="text">
        <string>Tea</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_other">
       <property name="text">
        <string>Other</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <widget class="QLabel" name="label_breakfast">
       <property name="text">
        <string>Breakfast</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QLineEdit" name="line_other"/>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="push_br2">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="push_lunch">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="push_tea">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="push_dinner">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="6" column="2">
      <widget class="QPushButton" name="push_other">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>push_br</tabstop>
  <tabstop>push_br2</tabstop>
  <tabstop>push_lunch</tabstop>
  <tabstop>push_tea</tabstop>
  <tabstop>push_dinner</tabstop>
  <tabstop>line_other</tabstop>
  <tabstop>push_other</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
